src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -27,17 +31,21 @@ class Lambda extends RequestStreamHandler {
   override def handleRequest(inputStream: InputStream, output: OutputStream, context: Context): Unit = {
     val inputString = inputStream.readAllBytes().map(_.toChar).mkString
     val input = read[Input](inputString)
+    val batchRef = input.executionId.split("-").take(3).mkString("-")
     for {
-      config <- configIo
+      _ <- logger.info(Map("batchRef" -> batchRef))(s"Starting workflow ${input.workflowContextName} for $batchRef")
```

Comment:
1. Could `Map("batchRef" -> batchRef)` be saved to a variable 
2. alternatively, do you think it's worth adding `logger.info` to a private method (like `logLambdaError`), so

```
def log(batchRef: String, msg: String) = logger.info(Map("batchRef" -> batchRef))(msg)
?

Reply:
I've changed it to how the others are. I've added 
```scala
val log = logger.info(Map("batchRef" -> batchRef))(_)
```
Then I can just call `log("Some message") 

On an unrelated note, I've also changed this to get the lambda name directly from the environment variables for the logger. I've used `sys.env` rather than the application.conf because it simplifies things a bit,